app/src/main/res/layout/content_event.xml

```diff
@@ -128,39 +127,54 @@
                 android:layout_height="wrap_content"
                 android:layout_marginLeft="@dimen/layout_margin_large"
                 android:layout_marginRight="@dimen/layout_margin_large"
-                android:layout_marginBottom="8dp"
-                android:text="About Event "
-                android:textSize="20dp" />
+                android:layout_marginBottom="@dimen/layout_margin_medium"
+                android:text="@string/about_event"
+                android:textColor="@color/black"
+                android:textSize="@dimen/event_details_headers" />
 
             <TextView
                 android:id="@+id/eventDescription"
                 android:layout_width="match_parent"
                 android:layout_marginLeft="@dimen/layout_margin_large"
                 android:layout_marginRight="@dimen/layout_margin_large"
                 android:layout_height="wrap_content"
-                android:lines="3"
+                android:lines="4"
```

Comment:
why is lines 3 here and int LINE_COUNT=3 above?

Reply:
It is 4

Reply:
why this is 4 here and LINE_count is 3 above?

Reply:
Actually the condition is more than 3 :+1:  not equal to 3

Reply:
okay